docs/en/status/query_ttl_setup.md

```diff
@@ -0,0 +1,29 @@
+# Get Effective TTL Configurations
+
+Time To Live (TTL) mechanism has different behaviors according to different storage implementations. By default, the
+core module provides two TTL configurations: [`recordDataTTL` and `metricsDataTTL`](../setup/backend/ttl.md).
+But every storage implementation could override these settings and provide its own TTL configurations, for example,
+BanyanDB provides its native TTL mechanism to support [progressive TTL](../banyandb/ttl.md) feature.
+
+This API is used to get the unified and effective TTL configurations.
+- URL, `http://{core restHost}:{core restPort}/status/config/ttl`
+- HTTP GET method.
+
+```shell
+> curl http://127.0.0.1:12800/status/config/ttl
+# Metrics TTL includes the definition of the TTL of the metrics-ish data in the storage,
+# e.g.
+# 1. The metadata of the service, instance, endpoint, topology map, etc.
+# 2. Generated metrics data from OAL and MAL engines.
+#
+# TTLs for each granularity metrics are listed separately.
+metricsTTL.minute=7
+metricsTTL.hour=7
+metricsTTL.day=7
+
+# Records TTL includes the definition of the TTL of the records data in the storage,
+# Records include traces, logs, sampled slow SQL statements, HTTP requests(by Rover), alarms, etc.
+# Super dataset of records are traces and logs, which volume should be much larger.
+recordsTTL.recordTTL=3
+recordsTTL.recordSuperDatasetTTL=3
```

Comment:
If I understand correctly this API should be consumed by BanyanDB, but from the response it looks like to be debugging or documenting purpose, it’s somewhat unclear to me what purpose it mainly focuses on, if it’s designed for BanyanDB to consume, maybe use a structured response and remove those comments, e.g. json, for easier decoding,

Reply:
> If I understand correctly this API should be consumed by BanyanDB

I just use local H2 to testing. not from BanyanDB.

This API is for adaptive and unified, not for BanyanDB.

Reply:
This is just a document to show the response. If we want to return for a JSON or YAML, I could support through `/status/config/ttl?format=json`.

Reply:
> > If I understand correctly this API should be consumed by BanyanDB
> 
> I just use local H2 to testing. not from BanyanDB.
> 
> This API is for adaptive and unified, not for BanyanDB.

I meant this API will be used by BanyanDB server, is it? If so, I think we can consider a more decoding-friendly format such as json, parsing json would be more convenient and less error prune than parsing `xx=yy` with comments `# comment`

Reply:
> I meant this API will be used by BanyanDB server, is it?

No, it isn't. This API is exposed to 3rd party. BanyanDB only accept its native gRPC protocol.
I am totally fine with a JSON output option, but I am not following your asking about `used by BanyanDB server`